.github/workflows/deploy.yml

```diff
@@ -0,0 +1,72 @@
+name: Build and Publish
+
+on:
+  push:
+    branches:
+      - 'main'
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.4
+        with:
+          persist-credentials: false
+
+      - name: Install and Build 🔧
+        uses: actions/setup-node@v2.1.5
+      - run: npm ci
+      - run: npm run test
+
+      - name: Set up QEMU 🔧
```

Comment:
why dont we publish to github, as tested here: https://github.com/klips-project/ghcr-ci-getstarted/tree/main/.github/workflows

Reply:
Removed QEMU. It is used to build against other platforms, which is not needed here yet.